docs/components/motor/dmc4000.md

```diff
@@ -15,17 +15,16 @@ When using it, you do not need to configure a [board component](/components/boar
 The DMC-40x0 controller can drive a variety of motor types, but the built-in Viam implementation supports only stepper motors at this time.
 You can drive other types of motors with Viam and the DMC-40x0 controller by [creating a custom motor model](/program/extend/).
 
-## Configuration
-
-Assign your motor a `name` (to identify the motor), `type` (`motor`) and `model` (`DMC4000`), and fill in the [attributes](#required-attributes) that apply to your particular hardware.
-Refer to your motor and motor driver data sheets for specifics.

```

Comment:
I think adding this line near the attributes table is helpful

Reply:
Kk, added 21 back in under attributes